tasks/drupal/deploy.yml

```diff
@@ -0,0 +1,187 @@
+  register: drupal_settings
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Customize drupal settings
+  blockinfile:
+    dest: "{{ steamengine_conf_path }}/settings.php"
+    block: "{{ steamengine_drupal_project_setting }}"
+    mode: u=rwx,g=rx,o=
+    owner: "{{ steamengine_project_user }}"
+    group: "{{ steamengine_app_user }}"
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Check www directory exists
+  stat:
+    path: "{{ steamengine_project_root_path_web }}"
+  register: root_path_web_directory
+
+- name: Fix permissions before updating source
+  command: "/{{ steamengine_project_name }}/bin/steamengine fixpermissions"
+  when: new_build_to_deploy and root_path_web_directory.stat.exists
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Clean www directory
+  file:
+    path: "{{ steamengine_project_root_path_web }}"
+    state: absent
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: "Move archive files to {{ steamengine_project_root_path_web }}"
+  command: "mv {{ tempdir.path }} {{ steamengine_project_root_path_web }}"
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: "Add read permission for {{ steamengine_app_user }}"
+  file:
+    path: "{{ steamengine_project_root_path_web }}"
+    owner: "{{ steamengine_project_user }}"
+    group: "{{ steamengine_app_user }}"
+    recurse: true
+    mode: u=rwx,g=rx,o=
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Clean configuration file
+  file:
+    state: absent
+    path: "{{ steamengine_project_root_path_web }}/{{ steamengine_drupal_public_directory }}/sites/default/settings.php"
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Create configuration link
+  file:
+    state: link
+    src: "{{ steamengine_conf_path }}/settings.php"
+    dest: "{{ steamengine_project_root_path_web }}/{{ steamengine_drupal_public_directory }}/sites/default/settings.php"
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+# Add .env file for env_variables drupal modules: https://www.drupal.org/project/env_variables
+
+- include_tasks:
+    file: "./deploy_env_file.yml"
+    apply:
+      tags:
+        - steamengine_deploy
+        - steamengine_deploy_drupal
+  when:
+    - steamengine_drupal_env is defined and steamengine_drupal_env
+    - new_build_to_deploy
+  tags:
+    - always
+
+- name: Check if database is empty
+  command: "{{ steamengine_drupal_drush_launcher_path }} -r {{ steamengine_project_root_path_web }} sql-query 'show tables'"
+  register: database_table_number
+  when: new_build_to_deploy
+  tags:
+    - steamengine_deploy_drupal
+
+- name: "Is a new deploy: {{ database_table_number.changed and not database_table_number.stdout }}"
+  set_fact:
+    initial_drupal_install: "{{ database_table_number.changed and not database_table_number.stdout }}"
```

Comment:
As there is no namespace, prefix your variables by "steamengine_drupal_".

Reply:
Should be fixed PR in https://github.com/STEAMULO/steamulo.steamengine/pull/33